soccer/gameplay/plays/restarts/basic_corner_kick.py

```diff
@@ -0,0 +1,62 @@
+import main
+import robocup
+import behavior
+import constants
+
+import standard_play
+import evaluation
+import tactics.coordinated_pass
+
+#Holds a point or a line, with the associted priority ranging from 0 to the number of lines/pts - 1 (0 is highest priority)
+class PointAndPriority:
+	def __init__(self, line_pt, priority):
+		self.line_pt = line_pt
+		self.priority = priority
+
+class BasicCorner(standard_play.StandardPlay):
+	def __init__(self):
+		super().__init__(continuous = True)
+		#point with lowest probability will have its probability scaled by MINIMUM_MULTIPLIER
+		MINIMUM_MULTIPLIER = 0.5
+		reflector = 1
+		#Depending on corner's location the points are refelcted
+		if main.ball().pos.x < 0:
+			reflector = -1
+
+		line1 = PointAndPriority(robocup.Line(robocup.Point(constants.Field.Width / -2 * 
+			0.633 * reflector, 7.45), 
+			robocup.Point(constants.Field.Width / -2 * 0.633 * reflector, 8.5)), 0)
+		line2 = PointAndPriority(robocup.Line(robocup.Point(constants.Field.Width / -2 
+			* 0.187 * reflector, 7.11), 
+			robocup.Point(constants.Field.Width / 2 * 0.147 * reflector, 6.2)), 1)
+		line3 = PointAndPriority(robocup.Line(robocup.Point(constants.Field.Width / 2 * 
+			0.8 * reflector, 5.85), 
+			robocup.Point(constants.Field.Width / 2 * 0.33 * reflector, 5.85)), 2)
```

Comment:
I would make these `/ -2 * 0.633` and the like constants.

Reply:
I would also make the priority a constant as well. It can either be an array or just line 1 priority etc

Reply:
I agree with Joe on this